ctru-rs/src/services/cfgu.rs

```diff
@@ -1,104 +1,224 @@
-//! Configuration service
+//! System Configuration service.
 //!
-//! This module contains basic methods to retrieve and change configuration from the console.
+//! This module contains basic methods to retrieve the console's system configuration.
 
 use crate::error::ResultCode;
 
+/// Console region.
+#[doc(alias = "CFG_Region")]
 #[derive(Copy, Clone, Debug, PartialEq, Eq)]
 #[repr(u32)]
 pub enum Region {
+    /// Japan.
     Japan = ctru_sys::CFG_REGION_JPN,
+    /// USA.
     USA = ctru_sys::CFG_REGION_USA,
+    /// Europe.
     Europe = ctru_sys::CFG_REGION_EUR,
+    /// Australia.
     Australia = ctru_sys::CFG_REGION_AUS,
+    /// China.
     China = ctru_sys::CFG_REGION_CHN,
+    /// Korea.
     Korea = ctru_sys::CFG_REGION_KOR,
+    /// Taiwan.
```

Comment:
Do these enum variants really need doc comments? It's nice to have everything documented but in this case it just seems redundant.

Reply:
I activated the lint for all docs since in most cases the enum variants have pretty useful documentation. This is just a small side effect that for me is worth having.

Tell me if you think there are better ways to handle these cases.

Reply:
I'd probably just add `#[allow(missing_docs)]` on enums like this, but up to you if you think that's better or not.